HyperVExtension/src/HyperVExtension/Models/VmCredentialAdaptiveCardSession.cs

```diff
@@ -53,7 +66,61 @@ void IExtensionAdaptiveCardSession.Dispose()
     public ProviderOperationResult Initialize(IExtensionAdaptiveCard extensionUI)
     {
         _extensionAdaptiveCard = extensionUI;
-        var operationResult = _extensionAdaptiveCard.Update(GetTemplate(), null, "VmCredential");
+        var showInfobar = _attemptNumber > 1;
+        int attemptNumberInText;
+        bool showOkButton;
+        string cancelText;

```

Comment:
NIT: Initialize strings to `string.empty`

Reply:
Then we'd need to change StyleCop rules to ignore "unnecessary assignment".

Reply:
Interesting.  Either StyleCop of the compiler understands that the strings will always be assigned a value.

To test remove the `else` block.  Then VS gives a compilation error about using un-assigned variables.

Interesting.